redpanda/templates/_helpers.tpl

```diff
@@ -230,13 +210,43 @@ IP is required for the advertised address.
 {{- end -}}
 
 {{- define "redpanda.pandaproxy.external.advertise.port" -}}
-{{- add1 (first .Values.config.pandaproxy.pandaproxy_api).port -}}
+{{- add1 (first .Values.listeners.rest.endpoints).port -}}
 {{- end -}}
 
 {{- define "redpanda.pandaproxy.external.advertise.nodeport.address" -}}
 {{- "$(HOST_IP)" -}}
 {{- end -}}
 
 {{- define "redpanda.pandaproxy.external.advertise.nodeport.port" -}}
-{{- add1 (first .Values.config.pandaproxy.pandaproxy_api).port -}}
+{{- add1 (first .Values.listeners.rest.endpoints).port -}}
+{{- end -}}
+
+{{/* ConfigMap variables */}}
+{{- define "admin-tls-enabled" -}}
+{{- $adminTlsEnabled1 := and .Values.auth.tls.enabled (or .Values.listeners.admin.tls.enabled (not (hasKey .Values.listeners.admin.tls "enabled"))) -}}
+{{- $adminTlsEnabled2 := and (not .Values.auth.tls.enabled) .Values.listeners.admin.tls.enabled -}}
+{{- print (and (or $adminTlsEnabled1 $adminTlsEnabled2) (not (empty .Values.listeners.admin.tls.cert))) -}}
+{{- end -}}
```

Comment:
This and the other `tls-enabled` templates return a string rather than a boolean, see this ticket: https://github.com/redpanda-data/helm-charts/issues/93